src/ripple/app/misc/ValidatorList.h

```diff
@@ -395,10 +403,13 @@ ValidatorList::onConsensusStart (
                 std::pair<std::size_t,PublicKey>(
                     std::numeric_limits<std::size_t>::max(), localPubKey_));
         }
-        // If no validations are being received, use all validators.
-        // Otherwise, do not use validators whose validations aren't being received
-        else if (seenValidators.empty() ||
-            seenValidators.find (val->first) != seenValidators.end ())
+        // If the total number of validators is too small, or
+        // no validations are being received, use all validators.
+        // Otherwise, do not use validators whose validations aren't
+        // being received.
+        else if (keyListings_.size() < MINIMUM_RESIZEABLE_UNL ||
```

Comment:
If you define `MINIMUM_RESIZEABLE_UNL` as 24, then I'm pretty sure you want `<=` here. I'd suggest simply setting `MINIMUM_RESIZEABLE_UNL` to be 25.